pkg/controller/tuned/tuned_controller.go

```diff
@@ -278,15 +294,44 @@ func createCustomResource(mgr manager.Manager) error {
 	return nil
 }
 
+func addOwnerReference(meta *metav1.ObjectMeta, tuned *tunedv1alpha1.Tuned) []metav1.OwnerReference {
+	var bController bool = false
```

Comment:
The boolean default zero value is already false. Why the name `bController`, maybe `isController`?

Reply:
Isn't explicit better than implicit?  isController sounds better, will change, thanks.  b == boolean